package.json

```diff
@@ -13,7 +13,6 @@
     "type": "git",
     "url": "https://github.com/lynnaloo/oran.git"
   },
-  "pre-commit":["test"],
```

Comment:
@troy0820 I did not delete pre-commit. I guess it wasnt there when I pulled this repo. I have pulled the changes from this repository. Please let me know if you have any other suggestions?